### PR TITLE
device: Changing HWg STE2 to YAML and adding new logos

### PR DESCRIPTION
**Pull request by FTBZ**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

screen shot 2017-11-14 at 16 55 53

### CODE REVIEW

**Comment by FTBZ** [comment]
Ping @crcro for information.

File: includes/definitions/hwg-ste2.yaml [hunk 1]
```diff
@@ -4,9 +4,37 @@ type: environment
 icon: hwg
 over:
     - { graph: device_temperature, text: Temperature }
-    - { graph: device_humidity, text: Humidity }
```
Comment (murrant) [inline includes/definitions/hwg-ste2.yaml]: Why was this overview graph removed?

Comment (FTBZ) [reply includes/definitions/hwg-ste2.yaml]: It's not a standard shipped sensor, only the temperature sensor is shipped by default. The humidity optional, so it's better to remove this graph as over.

**Comment by laf** [comment]
So I think this will lose historical data due to the index change. Whilst I prefer using more complete indexes now rather than single digits. In cases like this we should maintain historical data so just change to `index: '{{ $index }}'`

**Comment by FTBZ** [comment]
@laf Done, indexes were changed. But on my installation these sensors were not working so I don't think that will impact people.

File: includes/definitions/discovery/hwg-ste2.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                            op: '!='
+                            value: 4
+        state:
+            data:
+                -
+                    oid: inpTable
+                    value: inpAlarmState
+                    num_oid: .1.3.6.1.4.1.21796.4.9.1.1.4.
+                    descr: 'Alarm: {{ $inpName }}'
+                    index: '{{ $index }}'
```
Comment (laf) [inline includes/definitions/discovery/hwg-ste2.yaml]: Sorry you didn't need to do it here as that's a new sensor.

**Pull request merged**